libs/libcommon/include/common/avx2_mem_utils.h

```diff
@@ -150,42 +155,64 @@ FLATTEN_INLINE_PURE static inline bool check_block32x4_eq(const char * a, const
 
 FLATTEN_INLINE_PURE static inline int cmp_block32x4(const char * a, const char * b)
 {
-    if (check_block32x4_eq(a, b))
+    if (likely(check_block32x4_eq(a, b)))
         return 0;
-    for (size_t i = 0; i < AVX2_UNROLL_NUM - 1; ++i)
+    for (size_t i = 0; i < 3; ++i)
```

Comment:
why replace `AVX2_UNROLL_NUM - 1` with `3` here?

Reply:
The name of this function is `cmp_block32x4`, so I use exact number to make it clear.

Reply:
However, I think the previous way is better. I will rollback it .

Reply:
Fixed.